test-case/verify-sof-firmware-load.sh

```diff
@@ -13,25 +13,28 @@
 ##
 
 # source from the relative path of current folder
-source $(dirname ${BASH_SOURCE[0]})/../case-lib/lib.sh
+# shellcheck disable=SC1090
+source "$(dirname "${BASH_SOURCE[0]}")/../case-lib/lib.sh"
```

Comment:
Did you try the example on https://github.com/koalaman/shellcheck/wiki/SC1090 ?
Something like:
```shell
# shellcheck source=../case-lib/lib.sh
source "$(dirname "${BASH_SOURCE[0]}")/../case-lib/lib.sh"
```

Reply:
I already try this format, it report SC1091 :(

Reply:
I got `source=` working. I think it requires all three things:
1. `# shellcheck source=case-lib/lib.sh`
2. Run shellcheck from git top (just like Travis does)
3. Run shellcheck with -x (please submit the patch to add -x in travis)

Reply:
here refer the edit configure: vscode + shellcheck plugin  
configure: ` "shellcheck.customArgs": [ "-x" ] ` for this